1.3 Cart.feature

```diff
@@ -0,0 +1,33 @@
+Feature: 1.3 Cart Feature.
```

Comment:
It seems to me that you were inattentively reading the task - what exactly and in what quantity it was necessary to test